sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 5.0.1 (Unreleased)
+## 5.1.0-preview.1 (Unreleased)
```

Comment:
there should also be an entry about how the SDK targets service version 3.1-preview.1 instead of 3.0.0

Reply:
done.